common/tests/FileUtilsTest.java

```diff
@@ -543,15 +542,14 @@ public void testReadPrototxt() throws IOException {
     Files.createDirectories(testPath.getParent());
     Files.write(
         testPath,
-        ImmutableList.of(
-            "int32_field: 123",
-            "string_field: \"foo\"",
-            "map_field {",
-            "key: \"foo\"",
-            "value: 123",
-            "}",
-            "enum_field: YES"),
-        UTF_8);
+        ("int32_field: 123\n"
+                + "string_field: \"foo\"\n"
+                + "map_field {\n"
+                + "key: \"foo\"\n"
+                + "value: 123\n"
+                + "}\n"
+                + "enum_field: YES")
```

Comment:
Data which is written to the prototxt file in `testReadPrototxt()` method:
```
Files.write(
        testPath,
        ("int32_field: 123\n"
                + "string_field: \"foo\"\n"
                + "map_field {\n"
                + "key: \"foo\"\n"
                + "value: 123\n"
                + "}\n"
                + "enum_field: YES")
            .getBytes(UTF_8));
```
and `expected` result in `testWritePrototxt()` method:

```
String expected =
        "int32_field: 123\n"
            + "string_field: \"foo\"\n"
            + "map_field {\n"
            + "  key: \"foo\"\n"
            + "  value: 123\n"
            + "}\n"
            + "enum_field: YES\n";
```
are identical. Difference only in last `\n` symbol. I think will be better if `\n` symbol will be added to data in `testReadPrototxt()` method.

Reply:
I think it's easier to understand each test as it is now. If we introduce a common  `static final`, you won't see the string and the rest of the code on the screen at the same time.